src/game.ts

```diff
@@ -16,44 +16,26 @@ export type Providers = {
 export default class Game {
   private static games: Game[];
 
-  /** The internal name of the game. */
-  public name: string;
-  /** The aliases the game uses. */
-  public aliases: string[];
-  /** The human-formatted label of the game. */
-  public label: string;
-  /** The color representing the game. */
-  public color: string;
-  /** The game icon. */
-  public icon: string;
-  /** The game providers. */
-  public providers: Providers;
-  /** The Telegram IV templates. */
-  public telegramIVTemplates: TelegramIVTemplate[];
-
-  /** Creates a new Game.
-   *
-   * @param  {string} name - The internal name of the game.
-   * @param  {string[]} aliases - The aliases the game uses.
-   * @param  {string} label - The human-formatted label of the game.
+  /**
+   * Creates an instance of Game.
+   * @param {string} name The internal name of the game.
+   * @param {string[]} aliases The aliases the game uses.
+   * @param {string} label The human-formatted label of the game.
+   * @param {string} color The color representing the game.
+   * @param {string} icon The game icon.
+   * @param {Providers} providers The game providers.
+   * @param {TelegramIVTemplate[]} telegramIVTemplates The Telegram IV templates.
```

Comment:
I don't think we need explicit types here, they are derived from the constructor types